go.mod

```diff
@@ -13,8 +13,9 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
 	github.com/jawher/mow.cli v1.1.0 // indirect
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/crypto v0.0.0-20220112180741-5e0467b6c7ce
+	github.com/lestrrat-go/jwx v1.2.25
```

Comment:
assuming we are okay to use this library, lets use v2 if possible, to avoid tech debt in future